rootfs/builder/build.sh

```diff
@@ -202,8 +202,8 @@ if [[ "$slug_file" != "-" ]]; then
 					mc --quiet config host add $domain $keyID $secretKey
 					mc --quiet cp $slug_file $put_url/
 				fi
-			fi
-		else
-      curl -0 -s -o /dev/null -X PUT -T $slug_file "$put_url"
-  	fi
+			else
+      	curl -0 -s -o /dev/null -X PUT -T $slug_file "$put_url"
```

Comment:
My bad Its under else but the indentation was wrong. Some how indendation got messed up. I'll fix it ASAP